src/App.tsx

```diff
@@ -137,14 +141,13 @@ export default function App() {
 	const togglePlaying = useCallback(() => {
 		setPlaying(current => {
 			if(current) return false;
-
-			if(frameRef.current >= markerFileData.frames.length - 1) setFrame(frameStart); // restart if at end
+			if(checked == true && frameRef.current >= markerFileData.frames.length - 1) setFrame(frameStart); // restart if at end
```

Comment:
not sure the checkbox should affect this, since if we don't reset then it won't play anything

Reply:
right I forgot to remove it, was testing some things

Reply:
taken out